fs/arch/install-to-emmc-middle

```diff
@@ -0,0 +1,17 @@
+# start of install-to-emmc-middle
+
+export SYSTEMD=1
+export EUDEV=0
+
+# Arch is annoying, like holy moly. I need to mess with the lib folder again
+# ALSO I need hax like frick
+wget http://os.archlinuxarm.org/os/ArchLinuxARM-aarch64-latest.tar.gz
+mkdir /mnt/mv-then-del
+tar xfp ArchLinuxARM-aarch64-latest.tar.gz -C /mnt/mv-then-del
+rm -r /mnt/lib
+rm -r /mnt/mv-then-del/lib
+cp -r /lib /mnt/lib
+mv /mnt/mv-then-del/* /mnt/
+rm -r /mnt/mv-then-del
```

Comment:
`bsdtar xf ArchLinuxARM-aarch64-latest.tar.gz -C /mnt` why this doesn't work?

Reply:
Because you want to keep permissions:

  -p    Restore permissions (including ACLs, owner, file flags)

Reply:
ack, why are libraries shuffled around?

Reply:
That is from @FlameKat53 . Please see install-to-here as that's the only script I tested. Also see the discord.